### DISCUSSION TITLE
Trying to generate a modified version of the fonts using Glyphs

### DISCUSSION THREAD

**Discussion by gersiorll** [adobe-fonts/source-han-sans#453] [General]
I’m trying to modify a couple of characters using Glyphs. But when I try to export the font, I get an error message because the font contains glyphs with repeated names. These glyphs have nothing to do with the characters I’m editing. Apparently the Korean glyphs are causing the ones with repeated names. Given they’re thousands, it would take me ages to solve the problem manually. I'm trying to find an efficient way to solve this issue or would appreaciate if someone could point me on the right direction if Glyphs is not the appropiate sofware to make modifications of these fonts.

Thanks in advance

**Comment by CoolMarvel43** [comment]
Well this reply is for organizational purposes, and I am simply repeating myself here, but I have provided the appropriate links in my reply for anyone to look at.

As discussed in #119 and #292, I am hoping that Adobe will provide detailed and clear instructions on how to fork the project more easily, by just modifying the ExtraLight and Heavy masters and adding and removing glyphs as needed.

Also, I will quote part of a thread from #182, which was from 2018, when Dr. Ken Lunde was still the maintainer of this project:

> **For the long-term, we plan to make available sources that are easier to work with, meaning that new glyphs can be added and existing glyphs can be modified by touching only the masters. For the short-term, don't hold your breath. A lot of planning and preparation is needed to make this a reality, given the huge scope of these projects.**
> 
> _Originally posted by @/kenlunde in https://github.com/adobe-fonts/source-han-sans/issues/182#issuecomment-434139275_

**EDIT:** I removed some unnecessary text, as they do not concern the scope of this issue. I also marked in bold the appropriate quote by Ken Lunde.
